src/components/UserInformation.js

```diff
@@ -144,7 +154,7 @@ class UserInformation extends React.Component {
         <ul className='form-info__details'>
           <h2>User Details</h2>
           <li className='form-info__detail'>
-            <label>ACCESS Username</label>
+            <label>ACCESS Username {this.isFieldRequired('usernameOption') ? '*' : ''}</label>
```

Comment:
could add a separate function to annotate so you don't have to repeat this, maybe like
```js
const annotateRequired = (field, text) => this.isFieldRequired(field) ? text + "*" : text
// ...
<label>{annotateRequired('usernameOption', 'ACCESS Username')</label>
```
what do you think?